FishingTrawler.cpp

```diff
@@ -0,0 +1,67 @@
+#include "FishingTrawler.h"
+
+FishingTrawler::FishingTrawler() : numFishingNets(5), numCrewMembers(4), amountOfFishCaughtPerDay(323.4f)
+{
```

Comment:
Instead of initializing these members here, call the other constructor that takes arguments and use that to initialize the members.  
This concept is known as a Delegating Constructor:
```
Foo() : Foo(3, 4, 5) { }  //this ctor calls the other ctor that takes params
Foo(int, int, int) : /* etc */ { }
```
- [ ] revise